### PR TITLE
[Bug 1132052] Only suggest solved questions in suggestion API.

### PR DESCRIPTION
**Pull request by mythmon**
r?

### CODE REVIEW

File: kitsune/search/api.py [hunk 1]
```diff
@@ -54,7 +54,8 @@ def _question_suggestions(searcher, text, locale, product, max_results):
         model='questions_question',
         question_is_archived=False,
         question_is_locked=False,
-        question_has_helpful=True)
+        question_has_helpful=True,
```
Comment (rlr) [inline kitsune/search/api.py]: should we remove this has_helpful filter? I'm not sure if it's possible that an a question gets solved but doesn't have any answers marked helpful.

Comment (atopal) [inline kitsune/search/api.py]: I see no upside to filter for solved threads that also have helpful votes, so we should get rid of this filter.

**Comment by mythmon** [comment]
The Travis failure is because I forgot to update the tests. Fix coming.

**Comment by mythmon** [comment]
Fixes ^

**Comment by rlr** [comment]
:hot: :rocket:

**Pull request merged**